src/plugins/analysis/software_components/signatures/http_server.yara

```diff
@@ -36,3 +36,18 @@ rule nginx
 	condition:
 		$a and no_text_file
 }
+
+rule eCosWebServer
+{
+	meta:
+		software_name = "eCos Embedded Web Server"
+		open_source = true
+		website = "https://www.ecoscentric.com"
+		description = "Web-Server"
+		format_string = true
+	strings:
+		$a = /eCos Embedded Web Server/ nocase ascii wide
```

Comment:
`/.../` is used for regular expressions, right? So `"..."` would be better here i guess? (though it probably doesn't make a difference in the end)